.github/workflows/spelling.yml

```diff
@@ -0,0 +1,30 @@
+name: Spell checking
+on:
+  push:
+    branches:
+      - "**"
+    tags-ignore:
+      - "**"
+  pull_request_target:
+  issue_comment:
+    types: [created]
+
+jobs:
+  build:
+    name: Spell checker
+    runs-on: ubuntu-20.04
+    steps:
+    - name: checkout-merge
+      if: "contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2.0.0
+      with:
+        ref: refs/pull/${{github.event.pull_request.number}}/merge
+    - name: checkout
+      if: "!contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2.0.0
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+    - name: check-spelling
+      uses: ./
```

Comment:
Well, you are telling to find the Action in the current directory but you should add something like `check-spelling/check-spelling@v0.0.19` or `check-spelling/check-spelling@v0` (which looks a bit weird, not sure if specifying only major version works for unstable semver)